deploy-service/common/src/main/java/com/pinterest/deployservice/bean/EnvType.java

```diff
@@ -24,13 +24,19 @@
  *      Control stage type
  * CANARY:
  *      Canary stage type
-* LATEST:
+ * STAGING:
+ *    Staging stage type
+ * LATEST:
  *      LATEST stage type
+ * DEV:
+ *      DEV stage type
  */
 public enum EnvType {
     DEFAULT,
     PRODUCTION,
     CONTROL,
     CANARY,
-    LATEST;
+    STAGING,
+    LATEST,
+    DEV;
```

Comment:
Why we add more stage type dev and staging?

Reply:
Moreover, are these all the available types in PinDeploy?

Reply:
This was a apart of the slack conversation about the additional stage types. Yongwen had documented this in the PinDeploy Environment Type document in the "Teletraan Stage_Type" section. I won't link it here to not expose any links publicly but I will send it to you.